chromium_src/chrome/browser/profiles/profile_attributes_entry.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/profiles/profile_attributes_entry.h"
+
+void ProfileAttributesEntry::BraveMigrateObsoleteProfileAttributes() {
+  // Run our migrations
+#if !BUILDFLAG(IS_ANDROID)
+  constexpr int kPlaceholderAvatarIndex = 26;
+  constexpr int kBraveAvatarIconStartIndex = 56;
```

Comment:
`GetDefaultAvatarIconCount`?

Reply:
We can't use that because of this patch https://github.com/brave/brave-core/blob/master/patches/chrome-browser-profiles-profile_avatar_icon_util.cc.patch

Reply:
Found a way to do this properly! Fixed